drivers/pinctrl/pinctrl_smartbond.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022, Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/pinctrl.h>
+#include <DA1469xAB.h>
+
+/** Utility macro to retrieve starting mode register and pin count for GPIO port from DT */
+#define GPIO_PORT_ENTRY(nodelabel)							\
+		{ DT_REG_ADDR_BY_IDX(DT_NODELABEL(nodelabel), 1),			\
+		  DT_REG_SIZE_BY_IDX(DT_NODELABEL(nodelabel), 1) / sizeof(uint32_t) }
```

Comment:
I am not sure about it. For DA14695, the registers are there but the pins aren't due to package.
gpio node's `ngpios` property looks like a better option.